7_react/react1/week3/todo-list-app/src/EmployeesList.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+
+const Employees = () => {
+  const [data, setData] = useState([]);
+
+  const fetchData = async ()=>{
+
+    try {
+        let res = await fetch('https://gist.githubusercontent.com/benna100/5fd674171ea528d7cd1d504e9bb0ca6f/raw')
+        let data = await res.json()
+        setData(data)
+    }catch(error){
+        console.log("NOOOOOOOOOO")
+    }
+}
+    
+    useEffect(() => {
+        fetchData()
+    }, [])
+
+    const onDelete = (employeesName) => {
+        setData((previousEmployees) => previousEmployees.filter((Employees) => Employees.name !== employeesName))
```

Comment:
Good job on the correct use of filter. 
Just remember that in JavaScript, variable names always start with a small letter. So it should be `employees` instead of `Employees`.
This applies to most other languages as well, and is primarily to distinguish between variables and classes.